docs/changes/units/16139.api.rst

```diff
@@ -0,0 +1,2 @@
+Units now exposes ``get_converter`` which returns a function that
+can be used to convert scalar values directly between units.
```

Comment:
Not only scalar values; also perhaps tell when it would be useful:
```
Units now exposes ``get_converter`` which returns a function that
will convert a scalar or array from one unit to another. This can be
useful to speed up code that converts many quantities with the same
unit to another one, especially if the quantity has not many elements,
so that the overhead of creating a conversion function is relatively large.
```